test/ecs/components/MapPin.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { Engine, TextureFilterMode, TextureWrapMode, components } from '../../../packages/@dcl/ecs/src'
+import { testComponentSerialization } from './assertion'
+
+describe('Generated Map Pin ProtoBuf', () => {
+  it('should serialize/deserialize move Tween', () => {
+    const newEngine = Engine()
+    const MapPin = components.MapPin(newEngine)
+
+    testComponentSerialization(MapPin, {
+      title: 'Title',
+      description: 'Description',
+      iconSize: 0.5,
+      position: { x: 1, y: 2 },
+      texture: {
```

Comment:
are we sure about this ? Do we need all this fields or just a string to a path ?
@davidejensen